geopandas/geodataframe.py

```diff
@@ -58,6 +58,24 @@ class GeoDataFrame(GeoPandasBase, DataFrame):
     geometry : str or array (optional)
         If str, column to use as geometry. If array, will be set as 'geometry'
         column on GeoDataFrame.
+
+    Examples
+    --------
+    Constructing GeoDataFrame from a dictionary.
+
+    >>> d = {'col1': ['name1', 'name2'], 'geometry': [shapely.geometry.Point(1,2), shapely.geometry.Point(2,1)]}
+    >>> gdf = gpd.GeoDataFrame(d)
```

Comment:
```suggestion
    >>> d = {'col1': ['name1', 'name2'],
             'geometry': [shapely.geometry.Point(1,2), shapely.geometry.Point(2,1)]}
    >>> gdf = gpd.GeoDataFrame(d, crs="EPSG:4326")
```

Line break because flake8 does not like lines longer than 88 characters and added CRS. Even though CRS is not required, I'd rather have it here.